exonum-java-binding/service-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -117,9 +117,7 @@
         <version>2.22.2</version>
         <configuration>
           <argLine>
-            <!-- TODO: Replace with a path to the libjava_bindings.so to enable
-                 integration tests that require a native library. -->
-            -Djava.library.path=${project.basedir}/core/rust/target/debug
+            -Djava.library.path=${env.EXONUM_HOME}/lib/native
```

Comment:
Is this going to be used for integration tests? If so - shouldn't it point to `target/debug/deps`?